StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/PaymentMethodPicker/LinkPaymentMethodPicker.swift

```diff
@@ -50,7 +50,12 @@ final class LinkPaymentMethodPicker: UIView {
         }
     }
 
-    var supportedPaymentMethodTypes = Set(ConsumerPaymentDetails.DetailsType.allCases)
+    var supportedPaymentMethodTypes = Set(ConsumerPaymentDetails.DetailsType.allCases) {
+        didSet {
+            // TODO(tillh-stripe) Update this as soon as adding bank accounts is supported
+            addPaymentMethodButton.isHidden = !supportedPaymentMethodTypes.contains(.card)
```

Comment:
I noticed that pressing `Add payment method` in a bank-only session (`orderedPaymentMethodTypes == [.USBankAccount]`) does nothing, so let’s hide it, @davidme-stripe?

Reply:
Yep, it's not hooked up yet, I have a JIRA for this somewhere. We should plug it in once all this is working!